### PR TITLE
Move computeRTL to a separate file and add RTL support in state-info

### PR DESCRIPTION
**Pull request by rhayun**
i am not sure if the location of the compute_rtl.ts its in the context

### CODE REVIEW

**Comment by rhayun** [comment]
Travis failed, where should i move the import line?

**Review by zsarnett — COMMENTED** [review]
Try line 15 or above

File: src/common/util/compute_rtl.ts [hunk 1]
```diff
@@ -0,0 +1,9 @@
+import { HomeAssistant } from "../../types";
+
+export default function computeRTL(hass: HomeAssistant) {
```
Comment (balloob) [inline src/common/util/compute_rtl.ts]: Please don't use `export default`

```suggestion
export computeRTL = (hass: HomeAssistant) => {
```

File: src/layouts/home-assistant-main.js [hunk 2]
```diff
@@ -176,11 +177,7 @@ class HomeAssistantMain extends NavigateMixin(EventsMixin(PolymerElement)) {
   }
 
   computeRTL(hass) {
```
Comment (zsarnett) [inline src/layouts/home-assistant-main.js]: `_computeRTL` This shows its a private function and differentiates it from the `computeRTL` function

File: src/state-summary/state-card-display.js [hunk 3]
```diff
@@ -63,6 +75,11 @@ class StateCardDisplay extends LocalizeMixin(PolymerElement) {
         type: Boolean,
         value: false,
       },
+      rtl: {
+        type: Boolean,
+        reflectToAttribute: true,
+        computed: "computeRTL(hass)",
```
Comment (balloob) [inline src/state-summary/state-card-display.js]: All method used privately by this class should be prefixed, so name it `_computeRTL`

**Review by balloob — APPROVED** [review]

**Comment by balloob** [comment]
Please next time start with a clean branch forked off our latest dev.

**Comment by rhayun** [comment]
@balloob i want to continue and fix more regarding RTL, one thing i notice is that some icons are missing - for example in RTL the chevron need to be chevron-left and not chevron-right - but chevron-left are missing from core - can you please add it?

**Pull request merged**